environments/icds-cas/app-processes.yml

```diff
@@ -1,6 +1,12 @@
 formplayer_memory: "8000m"
 gunicorn_workers_static_factor: 4
 
+celery_heartbeat_thresholds:
+  icds_aggregation_queue: 86400
```

Comment:
This seems quite large. Maybe a couple hours? The agg should never be delayed by a full day.